src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -2273,6 +2275,21 @@ private static void CheckParameterModifierMismatchMethodConversion(SyntaxNode sy
                 return;
             }
 
+            // For synthesized delegates with mismatched escape rules, report an error to prevent
+            // ref-safety holes across old and new rule boundaries.
+            // See https://github.com/dotnet/roslyn/issues/76087
```

Comment:
Code should be ideally commented enough to not need "external" issue links. This one seems superfluous.

```suggestion
```